juspay/hyperswitch
# feat(multitenancy): move users and tenants to global schema

## Type of Change



## Description

Users will be global and not tied to a tenant
Login page will be one and not customized. An option to switch tenants will be provided

https://github.com/juspay/hyperswitch-product/issues/50

### Additional Changes





## Motivation and Context

To establish tenancy in our application and start using the default tenant as 'Hyperswitch' and also offer a 'test' tenant in all environments

Users will be global and not tied to a tenant
Login page will be one and not customized. An option to switch tenants will be provided


## How did you test it?


1. Start router and control center and try creating new user account.
2. Check if the user is getting created and an entry added in public schema for the user
3. Try sign out from the control center and login again
4. Login should work as expected

## Checklist
